renderer/HardwareOpenGL.cpp

```diff
@@ -731,6 +735,83 @@ int opengl_Setup(oeApplication *app, int *width, int *height) {
 }
 #endif
 
+#if defined(WIN32)
```

Comment:
What's specific to Windows here ? Is it just that `set_flags` needs to be implemented for other platforms ?

Reply:
That code has a couple of Win32 specific calls, specifically GetSystemMetrics and AdjustWindowRect. It may be possible to generalize this with some more API in the platform specific stuff, but it may not be worth it if the jump to SDL is made